### ISSUE TITLE
About Day 1

### ISSUE THREAD

**Issue by gentlefress** [Avik-Jain/100-Days-Of-ML-Code#66]
What's the meaning of random_state=0 in train_test_split()？

**Comment by FX196** [comment]
It means the seed for the random number generator is set to 0. Setting the seed makes the results reproducible. https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html

**Comment by gentlefress** [comment]
> It means the seed for the random number generator is set to 0. Setting the seed makes the results reproducible. https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html

yeah,I know ,thank you.And do you have only 54 days?I can't see the rest
